src/utils/web3_tools.js

```diff
@@ -116,3 +117,11 @@ export function link(contract, name, library) {
   contract.bytecode = contract.bytecode.replace(pattern, address);
 }
 
+export async function getBalance(web3, address) {
+  return new BN(await web3.eth.getBalance(address));
+}
+
+export function maximalGasPrice() {
+  const GAS_LIMIT = '8000000';
+  return new BN(GAS_LIMIT).mul(new Web3().utils.toWei(new BN(config.defaultGasPrice), 'gwei'));
```

Comment:
Same here. For calculations it's ok to wrap into BN. But then the result should be converted back into string.

Reply:
⚠️  Magic number warning ⚠️ Why not use the DEFAULT_GAS? All transactions are performed with it by default.